packages/multi-select-combo-box/theme/material/vaadin-multi-select-combo-box-styles.js

```diff
@@ -29,6 +29,16 @@ const multiSelectComboBox = css`
     caret-color: var(--material-body-text-color) !important;
   }
 
+  /* Override input-container styles */
+  ::slotted([slot='chip']),
+  ::slotted([slot='overflow']) {
+    padding: 0 0.5rem;
+  }
```

Comment:
For Material theme, we only need to override padding:

https://github.com/vaadin/web-components/blob/6e04c8dbfdaf839275441cd6374b399aa0dcf5b9/packages/input-container/theme/material/vaadin-input-container-styles.js#L39-L41